modules/compliance-raw-tailored.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-advanced.adoc
+
+[id="compliance-raw-tailored_{context}"]
+= Using raw tailored profiles
+While the TailoredProfile CR enables the most common tailoring operations, the XCCDF standard allows even more flexibility in tailoring OpenSCAP profiles. In addition, if your organization has been using OpenScap previously, you may have an existing XCCDF tailoring file and can reuse it.
+
+The ComplianceSuite object contains an optional TailoringConfigMap attribute that you can point to a custom tailoring file. The value of the TailoringConfigMap attribute is a name of a ConfigMap which must contain a key called `tailoring.xml` and the value of this key is the tailoring contents.
+
+.Procedure
+. Create the ConfigMap from a file:
++
+[source,terminal]
+----
+$ oc create configmap my-scan-tailoring --from-file=tailoring.xml=/path/to/the/tailoringFile.xml
```

Comment:
can we improve this command with filename either including or excluding file path

Reply:
I think the `--from-file` works well here, as it is specifying the key and file path: 

``` 
--from-file=[]
Key file can be specified using its file path, in which case file basename will be used as configmap key, or optionally with a key and file path, in which case the given key will be used.  Specifying a directory will iterate each named file in the directory whose basename is a valid configmap key.

# Create a new configmap named my-config with specified keys instead of file basenames on disk
  oc create configmap my-config --from-file=key1=/path/to/bar/file1.txt --from-file=key2=/path/to/bar/file2.txt
```

https://www.mankier.com/1/oc-create-configmap

Reply:
Ok, got it.